### ISSUE TITLE
Website can't display images from sekai.best or the fandom wiki

### ISSUE THREAD

**Issue by stellaaash** [FireInsidE-fie/haruka_closet#1]
When using \<img> elements, linking to images from sekai.best or the Project Sekai fandom wiki returns either extremely low resolution images or no image at all.

The browser console reveals 404 errors or the message `A resource is blocked by OpaqueResponseBlocking, please check browser console for details.`

**Comment by stellaaash** [comment]
I will have to add previews of the images to the gallery. Clicking on these previews will open a new tab to the full size image.

The preview image will most likely be a square snapshot of the entire image, possibly pixelated to go for a retro aesthetic.

**Comment by stellaaash** [comment]
Website to be used for pixelation : https://pinetools.com/pixelate-effect-image

**Comment by stellaaash** [comment]
I started to add the new images today, links are still to be added.

Commit dae0044a4b02da5851d629dfe1f2dc4b99a8018e

**Comment by stellaaash** [comment]
Links have been added today ; I would like to make the images square and add a nice pixelart border somehow...

19f92248aac46caaf8543e47c8eb59f1e025abd8

**Comment by stellaaash** [comment]
Made the images squared today! They're also close ups now.

0f6db8e7d2dd039004058f3c204bb5391d8a8480

**Comment by stellaaash** [comment]
Since the topic of this issue is basically resolved, I'll open a new issue to make the gallery more visually appealing in the future. Ideas : 

- Images well organized in a 3 images wide grid
- Borders for the image (pixel art?)
- Add more images!
